sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -30,6 +31,39 @@
 # This module is experimental. No backwards-compatibility guarantees.
 
 
+class StatusServer(object):
+
+  def start(self, STATUS_HTTP_PORT=0, started_callback=None):
```

Comment:
Can you document the arguments in pydocs?

And why do we need these arguments:
`STATUS_HTTP_PORT` is really constant 0 for now to pick a random port. We can use a constant instead.
`started_callback` seems like it is just a test construct. Do we need it?

Reply:
This is mostly for completion of testing.
As the new server is started on a separate thread, I needed a future/callback to know when its started.